benchmark/common_utils.hpp

```diff
@@ -566,4 +566,34 @@ static inline void calc_avg_counters(benchmark::State& state) {
 }  // namespace utils
 }  // namespace blas_benchmark
 
+/** Registers benchmark for the float data type
+ * @see BLAS_REGISTER_BENCHMARK
+ */
+#define BLAS_REGISTER_BENCHMARK_FLOAT(register_benchmark, args, exPtr, \
```

Comment:
I can't see anywhere that this is called where `register_benchmark` isn't just `register_benchmark`. It's quite weird looking!

Reply:
TBH I was expecting feedback to go into the other direction, to do the same for the test macro :) Removed the parameter: https://github.com/codeplaysoftware/sycl-blas/pull/245/commits/ae90906486cd0ad960fcc538bc1edc6ad87e2552